Feature/create atmosphere class

jonesbmj opened the pull request.

jonesbmj committed:
Edited .gitignore file to include .idea files

PyCharm adds weird .idea files somewhere/somehow and we don't want
them.

jonesbmj committed:
Added a basic version of the Atmosphere class

Includes a basic example. Later on, will increase complexity of each parameter to better model real world environment.

jonesbmj committed:
Addressed comments, added functions within class

Modified parameters of Atmosphere class. Added function for density,
gravity, pressure, temperature, viscosity. Used simple constants for the
time being. Added main() function.

ericdrejza commented on atmosphere.py:
```diff
@@ -0,0 +1,75 @@
+    self.temperature = self.temperature(altitude)
+    self.viscosity = self.viscosity(altitude)
+
+  def density(self, altitude):
```
We should look to name our methods differently than our class' properties

ericdrejza commented on atmosphere.py:
```diff
@@ -0,0 +1,75 @@
+    """
+    calculates temperature
+    :param altitude:
+    :return:
```
We should probably note what we're returning here, by mentioning the return type and what it represents

jonesbmj committed:
Renamed functions for clarity.

ericdrejza commented on atmosphere.py:
```diff
@@ -0,0 +1,75 @@
+    self.gravity = self.calc_gravity(altitude)
+    self.pressure = self.calc_pressure(altitude)
+    self.temperature = self.calc_temperature(altitude)
+    self.viscosity = self. calc_viscosity(altitude)
```
```suggestion
    self.viscosity = self.calc_viscosity(altitude)
```
Remove extra space

ericdrejza commented on atmosphere.py:
```diff
@@ -0,0 +1,75 @@
+    """
+    calculates density
+    :param altitude:
+    :return:
```
Add return type and description

ericdrejza commented on atmosphere.py:
```diff
@@ -0,0 +1,75 @@
+    :param altitude:
+    :return:
+    """
+    T = np.arange(15, 0, -15/100)  # C
```
We'll want to return a float here

jonesbmj committed:
Finalized initial atmosphere class (simplified)

Determined that keeping track of instantaneous  state will be beneficial to calculating
next state of rocket. Altitude parameter will be passed from the rocket
class.
Added variable types in function definition.

Removed altitude property
